dockerfiles/init/manifests/che.env

```diff
@@ -111,6 +111,15 @@
 #     containers from within their Docker-powered workspace.
 #CHE_DOCKER_PRIVILEGED=false
 
+# Host SecurityOpt Parameters
+#     This parameter allows to specify text values for the docker "--security-opt" parameter
+#     or the HostConfig{SecurityOpt:[]} value of the docker machine API 
+#     (https://docs.docker.com/engine/api/v1.33/#operation/ContainerCreate) 
```

Comment:
We use 1.21 API by default for compatibility with older docker versions. Can you point to that version instead?

Reply:
fixed